pkg/kn/commands/namespaced_test.go

```diff
@@ -98,21 +103,8 @@ func TestGetNamespaceAllNamespacesNotDefined(t *testing.T) {
 	expectedNamespace := "test1"
 	testCmd.SetArgs([]string{"--namespace", expectedNamespace})
 	testCmd.Execute()
-	actualNamespace, err := GetNamespace(testCmd)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if actualNamespace != expectedNamespace {
-		t.Fatalf("Incorrect namespace retrieved: %v, expected: %v", actualNamespace, expectedNamespace)
-	}
-}
-
-// test with all-namespace flag not defined and no namespace given
-func TestGetNamespaceDefaultAllNamespacesNotDefined(t *testing.T) {
```

Comment:
I have removed this test case because of ambiguous purpose.

- `TestGetNamespaceDefaultAllNamespacesNotDefined` is very similar to `TestGetNamespaceDefault`.
- The only difference is whether `allNamespaceFlag` is enabled or not via `testCommandGenerator`.
- The removed `TestGetNamespaceDefaultAllNamespacesNotDefined` tests default namespace when `allNamespaceFlag` is enabled. (I think that this scenario is not happen and not necessary for test.)